packages/react-app/src/pages/manage/new/wrapping_faq.json

```diff
@@ -1,19 +1,23 @@
 [
   {
     "question": "What is this?",
-    "answer": "This interfaces exposes two deployments of the VendingMachine contract for wrapping NU or KEEP into T tokens. The VendingMachine smart contract converts NU to T using a fixed conversion ratio that will be available indefinitely. Existing NU stakers *cannot* use the VendingMachine contract, instead existing NU/KEEP stakers will be grandfathered into Threshold via special staking adapters. It is not necessary to keep your worker node up until until the instructions for Threshold staking are shared."
+    "answer": "This interfaces exposes two deployments of the VendingMachine contract for wrapping NU or KEEP into T tokens. The VendingMachine smart contract wraps NU into T using a fixed conversion ratio that will be available indefinitely. Existing NU stakers *cannot* use the VendingMachine contract, only liquid NU / KEEP tokens can be wrapped to T. Instead existing NU/KEEP stakers will be grandfathered into Threshold via special staking adapters. It is not necessary to keep your worker node up until until the instructions for Threshold staking are shared."
   },
   {
     "question": "What happened to my NU staking details?",
     "answer": "The NuCypher Network staking contract was upgraded to ensure that legacy NU stakes can interoperate with the Threshold Network. Previous staking management functions are no longer supported through the NuCypher CLI/UI. Read-only stake information is still available and will be displayed here in a near-future update."
   },
+  {
+    "question": "Can I wrap staked NU/KEEP?",
+    "answer": "No. Existing NU stakers *cannot* use the VendingMachine contract, only liquid NU / KEEP tokens can be wrapped to T. Instead existing NU/KEEP stakers will be grandfathered into Threshold via special staking adapters."
+  },
   {
     "question": "What are the mainnet conversion ratios for NU and KEEP into T?",
     "answer": "The ratio for 1 NU is 3.259242493160745 T.  The ratio for 1 KEEP is 4.783188631255016 T."
   },
   {
     "question": "Once I have converted tokens into T can I unwrap them?",
-    "answer": "Yes, unwrapping NU is supported by the VendingMachine contracts will also be supported by this interface in a near-future update."
+    "answer": "Yes, unwrapping NU is supported by the VendingMachine contracts will also be supported by this interface and the official threshold dashboard in a near-future update."
```

Comment:
```suggestion
    "answer": "Yes, unwrapping NU / KEEP is supported by the VendingMachine contracts, and will be supported by this interface and the official Threshold dashboard in a near-future update."
```

Reply:
Were these suggestions committed before merge, or did we lose them?

Reply:
I don't believe so.

Reply:
Follow-up done in #161.